portal/app/[locale]/tunnel/_components/tunnelProviderToggle.tsx

```diff
@@ -69,29 +70,23 @@ export const TunnelProviderToggle = function ({
 
   return (
     <>
-      <div className="flex w-full items-center justify-center rounded-lg bg-neutral-100 p-1">
-        <button
-          className={`flex-1 rounded-md p-1 text-sm font-medium transition ${
-            providerType === 'native'
-              ? 'bg-white text-neutral-950 shadow-sm'
-              : 'text-neutral-600 hover:text-neutral-950'
-          }`}
-          onClick={toggleNativeProvider}
-          type="button"
-        >
-          {t('form.hemi-tunnel')}
-        </button>
-        <button
-          className={`flex-1 rounded-md p-1 text-sm font-medium transition ${
-            providerType === 'thirdParty'
-              ? 'bg-white text-neutral-950 shadow-sm'
-              : 'text-neutral-600 hover:text-neutral-950'
-          }`}
-          onClick={toggleThirdPartyProvider}
-          type="button"
-        >
-          {t('form.third-party-bridge')}
-        </button>
+      <div className="[&_li]:flex-1 [&_li]:md:flex-1 [&_ul.flex.w-full.flex-wrap.items-center.gap-x-2.gap-y-1]:gap-x-3 [&_ul.flex.w-full.flex-wrap.items-center.gap-x-2.gap-y-1_li]:py-0">
```

Comment:
This fairly large selector will probably break. If you want to target the direct tabs component container, which is the direct children of this `div`, maybe you can use


```suggestion
      <div className="[&_li]:flex-1 [&_li]:md:flex-1 [&>ul]:gap-x-3 [&>ul]:py-0">
```